addons/ammunition/CfgMagazines.hpp

```diff
@@ -2,40 +2,40 @@ class CfgMagazines {
     class CA_Magazine;
 
     CLASS(8Rnd_P_000): CA_Magazine {
-        author = "TyroneMF";
+        author = $STR_TACGT_Ammo_Author;
```

Comment:
This is pointless to stringtable, it's just names.

Reply:
Removed.